File: Python/6. 자릿수의 합 (중하)/AA.py
import sys
import math as m
#sys.stdin=open('input.txt', 'rt')

n=int(input())
arr=list(map(int,input().split()))

max=0
maxIndex=-1
for i in range(n):
    sum=0
    temp=arr[i]
    while temp>0:  
        sum+=temp%10
        temp=m.floor(temp/10)

    if max<sum:
        max=sum
        maxIndex=i

print(arr[maxIndex])

File: Javascript/섹션1. (완) 기본 문제 풀이/6. 홀수 (중)/AA.js
// A, B, C 세 수만을 입력 받는 문제
function solution(arr){
    let min=100

    let sum=0
    for(let i=0;i<arr.length;i++){
        if(arr[i]%2==1){
            sum+=arr[i]
            if(min>arr[i]){
                min=arr[i]
            }
        }
    }
    console.log(sum)

    return min
}
str="12 77 38 41 53 92 85"
arr=str.split(" ")
for(let i=0;i<arr.length;i++){
    arr[i]=parseInt(arr[i])
}
console.log(solution(arr))